salt/utils/network.py

```diff
@@ -46,18 +47,25 @@
     pass
 
 
-_INTERFACES = {}
+class Interfaces:
+    __slots__ = ("interfaces",)
 
+    def __init__(self, interfaces=None):
+        if interfaces is None:
+            interfaces = {}
```

Comment:
**suggestion (non-blocking)**: this *could* be `interfaces = interfaces or {}`, though if someone passes in an empty interfaces dict expecting it to get updated they could be surprised.

That being said, since `clear` is overwriting `self.interfaces` then that's probably not a huge surprise :thinking: